src/rpc/server.ts

```diff
@@ -898,4 +902,93 @@ export class Server {
     return jsonrpc.postObject(this.serverURL.toString(), 'getVersionInfo');
   }
 
+  /**
+   * Returns a contract's balance of a particular token, if any.
```

Comment:
for consideration, mention the contract balance is only applicable for SAC tokens, and link to https://developers.stellar.org/docs/tokens/stellar-asset-contract for reference.

Reply:
done in 8bc0e2de! good suggestion